data_store/table_extraction/config/pf_r1_config.py

```diff
@@ -12,7 +12,7 @@
 )
 from data_store.validation.pathfinders.schema_validation.consts import PFEnums
 
-PF_TABLE_CONFIG = {
+PF_TABLE_CONFIG: dict[int | str, dict[str, dict]] = {
```

Comment:
Definitely a starting attempt 😓

Reply:
It seems like every time some function expects a `dict[int | str, dict[str, dict]]` type and the dict keys are manually set to always be `str`, we have to type-hint it to include `int` as well, otherwise it will infer that its type is only `dict[str, dict[str, dict]]`. Sad.